test/auth_cluster/test_auth_v2_migration.py

```diff
@@ -70,6 +70,17 @@ async def populate_auth_v1_data(manager: ManagerClient):
     ])
 
 
+async def warmup_v1_static_values(manager: ManagerClient, hosts):
+    # auth-v1 was using statics to cache internal queries
+    # in auth-v2 those statics were removed but we want to
+    # verify that it was effective so trigger here internal
+    # call to potentially populate static storage and we'll
+    # verify later that after migration properly formed query
+    # executes (query has to change because keyspace name changes)
+    cql = manager.get_cql()
+    await asyncio.gather(*(cql.run_async("LIST ROLES", host=host) for host in hosts))
```

Comment:
we can't have black box here because if the switch didn't work we will both WRITE and later READ from v1 and this will pass despite being wrong behavior.

Or more precisely write to v1, migrate to v2, read again from v1 (for now we keep v1 data after migration)

Reply:
We could restart the node / cluster after migration.

So we could do something like:
- write
- migrate
- write again
- restart
- read and verify it sees both writes

with the `static`s, the read would not observe the second write, it would only see the first write.

Reply:
that would be more relaxed test, what if both writes are to v1, and read also from v1. This test wouldn't discover if migration is noop.

Reply:
Then maybe a test which somehow depends on the differences between auth-v2 and auth-v1. Like availability? I.e. auth-v2 is always accessible locally

Reply:
there is separate test for avaliability, why mix those things together? I think the current test is relatively simple and robust, it's better to be explicit when test for explicit things.

Reply:
So doing select is user visible because user can do selects on auth tables. It 's black box according you your definition. Although not sure why black box would be better than white box in this particular example. We anyway (ab)use our knowledge that there's problem with statics.